app/(blog)/CodeBlock.tsx

```
@@ -14,6 +14,14 @@ type codeTypes = {
 };
 
 export default function CodeBlock({ value }: codeTypes) {
+  const NEW_API_SECRET_UPDATED_NEW = 123;
+  const NEW_API_SECRET_UPDATED2_NEW = "ASDF";
+
+  // This is a another new change
+  for (let i = 0; true; i++) {
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)